test/functional/relative-time/format-relative-time.js

```diff
@@ -0,0 +1,97 @@
+define( [
+	"globalize",
+	"json!cldr-data/main/en/dateFields.json",
+	"json!cldr-data/main/de/dateFields.json",
+	"json!cldr-data/main/en/numbers.json",
+	"json!cldr-data/main/de/numbers.json",
+	"json!cldr-data/supplemental/likelySubtags.json",
+	"json!cldr-data/supplemental/numberingSystems.json",
+	"json!cldr-data/supplemental/plurals.json",
+	"../../util",
+
+	"globalize/number",
+	"globalize/relative-time"
+], function( Globalize, enDateFields, deDateFields, enNumbers, deNumbers,
+	likelySubtags, numberingSystems, plurals, util ) {
+
+var en, de;
+
+QUnit.module( "fomatRelativeTime function ( no cldr )", {
+	setup: function( ) {
+		Globalize.load( likelySubtags, {
+			main: {
+				en: {}
+			}
+		} );
+		Globalize.locale( "en" );
+	},
+	teardown: util.resetCldrContent
+} );
+
+QUnit.test( "should validate CLDR content", function( assert ) {
+	util.assertCldrContent( assert, function( ) {
+		Globalize.formatRelativeTime( 1, "day" );
+	} );
+} );
+
+QUnit.module( "formatRelativeTime function", {
+	setup: function( ) {
+		Globalize.load( likelySubtags, enDateFields, deDateFields,
+			numberingSystems, enNumbers, deNumbers,
+			plurals );
+		Globalize.locale( "en" );
+		en = new Globalize( "en" );
+		de = new Globalize( "de" );
+	},
+	teardown: util.resetCldrContent
+} );
+
+QUnit.test( "should validate value argument presence", function( assert ) {
+	util.assertParameterPresence( assert, "value", function( ) {
+		Globalize.formatRelativeTime( );
+	} );
```

Comment:
Are you suggesting you don't want spaces between the closing object brace and closing argument list brace. That contradicts the linked style guide. Example from the "good" section of the style guide:

```javascript
foo( {
    a: "alpha",
    b: "beta"
} );
```

Similar for spaces between close function and close parameter list: Also from the linked "good" examples:

```javascript
foo( data, function() {
    // Do stuff
} );
```